### PR TITLE
Updated order endpoints

### PR DESCRIPTION
**Pull request by ankitdas13**
Api doc reference 

[View RTO Reasons](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#12-view-rto-risk-reasons)

[Update fulfilment details](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#13-update-the-fulfillment-details)

### CODE REVIEW

File: documents/order.md [hunk 1]
```diff
@@ -272,7 +272,82 @@ client.order.edit(orderId,{
...
+}
+
+client.order.editFulfillment(orderId, request)
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
+| orderId* | string | The unique identifier of an order to access the fulfillment information. |
+| payment_method | string | The id of the order to be retrieve payment info |
```
Comment (razorpay-sanjib) [inline documents/order.md:329]: Is the `payment_method` description a proper one?

File: documents/order.md [hunk 2]
```diff
@@ -272,7 +272,82 @@ client.order.edit(orderId,{
...
+| shipping | array  | All keys listed [here](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#13-update-the-fulfillment-details) are supported |
```
Comment (razorpay-sanjib) [inline documents/order.md:330]: Is the link provided here a valid one?

**Review by razorpay-sanjib — CHANGES_REQUESTED** [review]

File: documents/order.md (hunk 1 above)
Comment (ankitdas13) [reply documents/order.md]: added all possible values for payment_method

File: documents/order.md (hunk 2 above)
Comment (ankitdas13) [reply documents/order.md]: update beta link to prod link

**Review by razorpay-sanjib — APPROVED** [review]
LGTM

**Review by razorpay-sanjib — APPROVED** [review]

**Pull request merged**